python/pyarrow/rados.py

```diff
@@ -24,12 +24,33 @@
 
 
 class SplittedParquetWriter(object):
+    """
+    Write data to a file of RadosParquetFile format.
+
+    Parameters
+    ---------
+    filename: The name of the file to write to.
+    destination: The destination of the file to write to.
```

Comment:
The directory where to write the split Parquet files.

Reply:
Done.